app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
toán tử `||=` là gì?
và tại sao ở đây lại phải sử dụng biến instance?

Reply:
a ||= b tương đương với a = a || b
sử dụng biến instance để giá trị lấy được tồn tại đến khi đối tượng bị hủy chứ không phải kết thúc hàm -> không phải lặp lại việc truy xuất dữ liệu